### DISCUSSION TITLE
Pain points?

### DISCUSSION THREAD

**Discussion by dsrw** [dsrw/enu#34] [General]
Over at https://forum.nim-lang.org/t/10700#71599, @moigagoo raised some excellent points about some things that could be easier, are non-obvious, or just plain don't work. Is there anything in Enu that's harder than it should be? (Answer: Yes) If so I'd love to hear about it here!

**Comment by viralpoetry** [comment]
Hi @dsrw , is there a known limit to the map size (I mean number of blocks)? I am trying to generate map file (json with voxels) from the heatmap but the program crashes if I do not restrict to few thousands voxels.
